Assets/Scripts/Framework/TargetSelector.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Framework
+{
+    public class TargetSelector : MonoBehaviour
+    {   
+        [SerializeField] private List<Transform> targetList;
+        [SerializeField] private LineRenderer lineRenderer;
+        [SerializeField] private Transform currentTarget;
+        [SerializeField] private float closeRange = 5.0f;
+
+        private int currentTargetIndex;
+
+        private void Start()
+        {
+            currentTargetIndex = 0;
+            currentTarget = targetList[1];
+            
+            lineRenderer.positionCount = 2;
```

Comment:
```suggestion
            lineRenderer.positionCount = targetList.Count;
```
Ik snap niet waarom de lijst maar 2 is.